src/scripts/launch

```diff
@@ -62,6 +63,7 @@ if [ -n "$precmd" ]; then
 fi
 cl $cl_parms "$cmd" 
 exitCode=$?
+cl "$text_command" 
```

Comment:
Doesn't this assume that the text_command is always set, but it is only set for SQL

Reply:
Do you have to have to check if text_command is set and assume it is not set if only 8 parms are passed